src/index.html

```diff
@@ -20,8 +21,523 @@
       href="styles/index.scss"
     />
   </head>
+  <!-- #endregion -->
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#home"
+        class="header__logo-link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="header__logo"
+        />
+      </a>
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-list-item">
+            <a
+              href="#Apple"
+              class="header__link header__link--is-active"
```

Comment:
Fix classes to pass failed tests
<img width="1510" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/64750958/e58e8505-7738-4487-b01f-ef8ef39c6797">
```suggestion
              class="nav__link is-active"
```